hack/verify/validate-sec.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+go get -u github.com/securego/gosec/cmd/gosec
```

Comment:
```suggestion
go get github.com/securego/gosec/cmd/gosec
```

Reply:
that should get `gosec` installed, your `validate-sec.sh` to run and the `make verify` pr test to run but fail because of the issues gosec discovers. At that point, we'll try to test the recommendations and if all is well the recommendations will get merged, we'll unhold this PR and rerun the tests - make verify should pass that time.

Reply:
thanks for the suggestion!